### PR TITLE
[Refactor] Redesign user verification email

### PR DESCRIPTION
**Pull request by zobweyt**
### Description
This pull request includes changes related to redesigning user email verification, formatting code and adding type annotations, fixing type warnings, and reordering configuration properties in alphabetical order.

### Changes
* Redesign user email verification (c26cb8f4f19a75fe5b2194879034bbde8de9ec21)
* Format code & Add type annotations (370ba90bcc53965a21bd34dc37fa34feb18f0055)
* Fix type warnings (f322c99b31e20f3be50bfc6e2a6afed1992c6414)
* Reorder configuration properties in alphabetical order (304cd50bdf793415840de97cf631ba00f38dd230)

### Related Issues
* Resolves project task [`70517894`](https://github.com/orgs/rapid-integration/projects/1/views/2?pane=issue&itemId=70517894)

### CODE REVIEW

File: backend/src/core/security.py [hunk 1]
```diff
@@ -10,18 +10,18 @@
...
 
 
 def decode_access_token(token: str) -> str | None:
     try:
-        decoded_token = jwt.decode(token, settings.SECRET_KEY, algorithms=settings.ALGORITHM)
-        data = JWTPayload(**decoded_token)
-        return data.sub
+        payload = jwt.decode(token, settings.SECRET_KEY, algorithms=[settings.ALGORITHM])
+        data = JWTPayload(**payload)
+        return str(data.sub)
```
Comment (Tanax-Xt) [inline backend/src/core/security.py:24]: why you use `str(data.sub)` here? data.sub can be not only str type

Comment (zobweyt) [reply backend/src/core/security.py:24]: it's required to return `str | None` in this function, but `data.sub` can be `int`. so type conversion is necessary

Comment (zobweyt) [reply backend/src/core/security.py:24]: otherwise it will cause in runtime error

Comment (Tanax-Xt) [reply backend/src/core/security.py:24]: ok

File: backend/src/emails/templates/verification-email.html [hunk 2]
```diff
@@ -1,43 +1,74 @@
...
-        <a href="{{ link }}" class="button">Verify Email</a>
-        <p class="link">If the button doesn't work, please <a href="{{ link }}">click here</a></p>
-        <p>This link will expire in {{ expires_at }} minutes.</p>
-    </div>
+  <img src="https://i.ibb.co/WtPCGfJ/Logo.png" alt="Logo" />
+  <h1>Please verify your email address</h1>
+  <p>You recently added a new email address to your Fusion profile. Please verify this email address by clicking the button below:</p>
+  <a href="{{ link }}" class="button">Verify Email</a>
+  <p>The link will expire in {{ expires_at }} minutes.</p>
+  <small>You are receiving this email because you have a Fusion account associated with <a href="mailto:mail@{{ email }}">{{ email }}</a>.</small>
```
Comment (Tanax-Xt) [inline backend/src/emails/templates/verification-email.html]: Can you put the content related specifically to email verification in a separate file in which you will inherit from the standard Fusion email template?

You can take an example from here https://jinja.palletsprojects.com/en/latest/templates/

Comment (zobweyt) [reply backend/src/emails/templates/verification-email.html]: it's not possible to use inheritance in plain HTML files

Comment (Tanax-Xt) [reply backend/src/emails/templates/verification-email.html]: Please study this topic in more detail. Perhaps you will find a solution.

**Pull request merged**